lib/cdo/google_drive.rb

```diff
@@ -94,6 +98,25 @@ def add_sheet_to_spreadsheet(data, spreadsheet_path, sheet_name)
       worksheet.save
     end
 
+    # Update a sheet (tab) in an _existing_ Google Sheets document, completely overwriting
+    # its contents, and creating it if needed.
```

Comment:
does it create the sheet if it doesn't exist or throw an error?

Reply:
Noticed the inconsistency a minute ago and just pushed a commit to correct this - it will create a sheet/tab (but not a document) if it doesn't exist.